Automattic/wp-calypso
# Stats: Rename existing highlight cards

Related to #72493

## Proposed Changes

* Renames "HighlightCards" to "WeeklyHighlightCards".
* Renames "HighlightCard" to "CountComparisonCard".

These changes are in preparation for a larger refactor for `CountComparisonCard` and `WeeklyHighlightCards`. It also establishes a naming convention for other card collections like `AllTimeHighlightCards` and `SubscriberHighlightCards` that currently exist outside of the components package.

## Testing Instructions



* Open the live branch for this PR.
* Ensure that the traffic and insights pages of Calypso Stats look and work as expected.

## Pre-merge Checklist
